provision/common/Taskfile.yaml

```diff
@@ -93,7 +95,8 @@ tasks:
           --arg kc_repo "{{ .KC_REPOSITORY }}" \
           --arg kc_branch "{{ .KC_BRANCH }}" \
           --argjson is_multi_az "{{ .MULTI_AZ }}" \
-          '. + { "numOfPods": ($num_of_pods|tonumber), "cpuLimitPerPod": ($cpu_limit|tonumber),
+          '. + { "numOfPods": ($num_of_pods|tonumber), "cpuRequestsPerPod": ($cpu_requests_per_pod|tonumber),
+                             "cpuLimitsPerPod": (if $cpu_limits_per_pod | length == 0 then null else ($cpu_limits_per_pod | tonumber?) end),
```

Comment:
can this be re-written as `(if ($cpu_limits_per_pod | length) == 0 then null else ($cpu_limits_per_pod | tonumber?) end)` @andyuk1986 ?